pennylane/devices/tests/test_compare_default_qubit.py

```diff
@@ -66,8 +67,13 @@ def circuit(theta, phi):
         grad_def = qml.grad(qnode_def, argnum=[0, 1])
         grad = qml.grad(qnode, argnum=[0, 1])
 
-        assert np.allclose(qnode(theta, phi), qnode_def(theta, phi), atol=tol(dev.shots))
-        assert np.allclose(grad(theta, phi), grad_def(theta, phi), atol=tol(dev.shots))
+        def workload():
+            return qnode(theta, phi), qnode_def(theta, phi), grad(theta, phi), grad_def(theta, phi)
+
+        qnode_res, qnode_def_res, grad_res, grad_def_res = benchmark(workload)
```

Comment:
If benchmarking is disabled here, does the `workload` function still run, or do we need to do something to make sure it does?

e.g. do we need `benchmark(workload)()` or similar?

Reply:
Workload will run seamlessly with benchmark disabled.